### DISCUSSION TITLE
希望有大佬能开发可以调整播放器大小的插件

### DISCUSSION THREAD

**Discussion by dbxsj** [the1812/Bilibili-Evolved#4950] [功能建议 / Ideas]
bilibili网页版的播放器实在是太小了看的人头晕眼花，希望有相应插件可以将播放器变大，类似于anfuns的播放器一样大就可以了

**Comment by the1812** [comment]
宽屏 / 网页全屏 / 全屏都可以变大

**Reply by dbxsj** [reply]
不行，这个不好用，有延迟，而且右侧没有推荐视频和up主信息了，我希望能在不改变原版布局基础上增大播放器的插件

**Reply by Geeyun-JY3** [reply]
在右侧保留推荐视频和 UP 主信息的前提下播放器真的还能增大吗……而且我个人实测 AcFun 的播放器在普通模式下也是比 bilibili 的小一圈啊……
印象中确实是有脚本提供过自定义播放器大小的功能，不过用例好像更多是缩小……
